# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1520723269 28800
#      Sat Mar 10 15:07:49 2018 -0800
# Node ID 7d66084d26604d49eb7516226fef346349788ca6
# Parent  6266e321ef22358a36f5a5fb0711ef85eeb441bd
maint: sort lists of function files in build system

* scripts/image/module.mk (%canon_reldir%_FCN_FILES): Sort the list.
* scripts/linear-algebra/module.mk (%canon_reldir%_FCN_FILES): Likewise.
* scripts/miscellaneous/module.mk (%canon_reldir%_FCN_FILES): Likewise.
* scripts/sparse/module.mk (%canon_reldir%_PRIVATE_FCN_FILES): Likewise.
* scripts/statistics/module.mk (%canon_reldir%_FCN_FILES): Likewise.

diff --git a/scripts/image/module.mk b/scripts/image/module.mk
--- a/scripts/image/module.mk
+++ b/scripts/image/module.mk
@@ -45,19 +45,19 @@ FCN_FILE_DIRS += \
   %reldir%/ind2rgb.m \
   %reldir%/iscolormap.m \
   %reldir%/jet.m \
   %reldir%/lines.m \
   %reldir%/ocean.m \
   %reldir%/pink.m \
   %reldir%/prism.m \
   %reldir%/rainbow.m \
+  %reldir%/rgb2gray.m \
   %reldir%/rgb2hsv.m \
   %reldir%/rgb2ind.m \
-  %reldir%/rgb2gray.m \
   %reldir%/rgbplot.m \
   %reldir%/spinmap.m \
   %reldir%/spring.m \
   %reldir%/summer.m \
   %reldir%/viridis.m \
   %reldir%/white.m \
   %reldir%/winter.m
 
diff --git a/scripts/linear-algebra/module.mk b/scripts/linear-algebra/module.mk
--- a/scripts/linear-algebra/module.mk
+++ b/scripts/linear-algebra/module.mk
@@ -15,18 +15,18 @@ scripts_linear_algebra_FCN_FILES = \
   %reldir%/isdefinite.m \
   %reldir%/isdiag.m \
   %reldir%/ishermitian.m \
   %reldir%/issymmetric.m \
   %reldir%/istril.m \
   %reldir%/istriu.m \
   %reldir%/krylov.m \
   %reldir%/linsolve.m \
+  %reldir%/logm.m \
   %reldir%/lscov.m \
-  %reldir%/logm.m \
   %reldir%/normest.m \
   %reldir%/normest1.m \
   %reldir%/null.m \
   %reldir%/ols.m \
   %reldir%/orth.m \
   %reldir%/planerot.m \
   %reldir%/qzhess.m \
   %reldir%/rank.m \
diff --git a/scripts/miscellaneous/module.mk b/scripts/miscellaneous/module.mk
--- a/scripts/miscellaneous/module.mk
+++ b/scripts/miscellaneous/module.mk
@@ -25,18 +25,18 @@ FCN_FILE_DIRS += \
   %reldir%/fileattrib.m \
   %reldir%/fileparts.m \
   %reldir%/fullfile.m \
   %reldir%/genvarname.m \
   %reldir%/getfield.m \
   %reldir%/grabcode.m \
   %reldir%/gunzip.m \
   %reldir%/info.m \
+  %reldir%/inputParser.m \
   %reldir%/inputname.m \
-  %reldir%/inputParser.m \
   %reldir%/isdeployed.m \
   %reldir%/isdir.m \
   %reldir%/ismac.m \
   %reldir%/ispc.m \
   %reldir%/isunix.m \
   %reldir%/license.m \
   %reldir%/list_primes.m \
   %reldir%/loadobj.m \
@@ -71,20 +71,20 @@ FCN_FILE_DIRS += \
   %reldir%/symvar.m \
   %reldir%/tar.m \
   %reldir%/tempdir.m \
   %reldir%/tmpnam.m \
   %reldir%/unix.m \
   %reldir%/unpack.m \
   %reldir%/untar.m \
   %reldir%/unzip.m \
+  %reldir%/validateattributes.m \
   %reldir%/ver.m \
   %reldir%/version.m \
   %reldir%/what.m \
-  %reldir%/validateattributes.m \
   %reldir%/zip.m
 
 %canon_reldir%dir = $(fcnfiledir)/miscellaneous
 
 %canon_reldir%_DATA = $(%canon_reldir%_FCN_FILES)
 
 %canon_reldir%_privatedir = $(fcnfiledir)/miscellaneous/private
 
diff --git a/scripts/sparse/module.mk b/scripts/sparse/module.mk
--- a/scripts/sparse/module.mk
+++ b/scripts/sparse/module.mk
@@ -1,16 +1,16 @@
 FCN_FILE_DIRS += \
   scripts/sparse \
   %reldir%/private
 
 %canon_reldir%_PRIVATE_FCN_FILES = \
-  %reldir%/private/__sprand__.m \
   %reldir%/private/__alltohandles__.m \
-  %reldir%/private/__default__input__.m
+  %reldir%/private/__default__input__.m \
+  %reldir%/private/__sprand__.m
 
 %canon_reldir%_FCN_FILES = \
   %reldir%/bicg.m \
   %reldir%/bicgstab.m \
   %reldir%/cgs.m \
   %reldir%/colperm.m \
   %reldir%/eigs.m \
   %reldir%/etreeplot.m \
diff --git a/scripts/statistics/module.mk b/scripts/statistics/module.mk
--- a/scripts/statistics/module.mk
+++ b/scripts/statistics/module.mk
@@ -1,15 +1,15 @@
 FCN_FILE_DIRS += scripts/statistics
 
 %canon_reldir%_FCN_FILES = \
   %reldir%/bounds.m \
   %reldir%/center.m \
+  %reldir%/corr.m \
   %reldir%/corrcoef.m \
-  %reldir%/corr.m \
   %reldir%/cov.m \
   %reldir%/discrete_cdf.m \
   %reldir%/discrete_inv.m \
   %reldir%/discrete_pdf.m \
   %reldir%/discrete_rnd.m \
   %reldir%/empirical_cdf.m \
   %reldir%/empirical_inv.m \
   %reldir%/empirical_pdf.m \
